mopidy_listenbrainz/listenbrainz.py

```diff
@@ -63,6 +63,21 @@ def get_requests_session(proxy_config, user_agent):
     return session
 
 
+class _RequestError(Exception):
```

Comment:
What if we made this a public exception and stored the message/status code in it, then we handled exceptions in front/backend? That way we can re-use this class.

Reply:
or have different handlers (e.g., ignore, different log level) based on when the exception occurs.

Reply:
My goal (not clearly expressed, sorry for that) was to have HTTP status in the logs in case of unexpected errors. This in order to be able to debug situations like: Token expiry, rate limits, ...

I find it more readable to keep HTTP notions with the HTTP client calls. But if there're good reasons to make the exception public, and propagate HTTP status away from the client calls, then why not. I don't see those reasons right now (I don't understand where such exceptions would be used in the frontend).

Reply:
there may be instances in the future (for example, with rate limiting) where we will want to know when a request fails so we can recover from it/do some other behavior; for instance, right now there would be no difference to the frontend whether an API request failed or the user just has no recommendation playlists.  do you think it's the right choice to leave this ability to recover just in the API client? i can't say I'm a huge fan of just returning an empty object to the consumer (frontend/backend) when a request fails, but if there's a good argument for doing this I'm open to reconsidering this.  I'd love to know what you think!